### PR TITLE
Wes/institution permissions

### PR DESCRIPTION
**Pull request by wesrisenmay-mx**

### CODE REVIEW

File: apps/institution-service/src/shared/utils/permissionValidation.test.ts [hunk 1]
```diff
@@ -231,4 +233,105 @@ describe("permissionValidation", () => {
       ).toBe(EditAggregatorIntegrationValidationErrorReason.NotYourAggregator);
     });
   });
+
+  describe("validateUserCanCreateAggregatorIntegration", () => {
+    it("returns true if they are a super admin and they there are aggregators without an integration", async () => {
```
Comment (Craiting) [inline apps/institution-service/src/shared/utils/permissionValidation.test.ts:238]:
```suggestion
    it("returns true if they are a super admin and there are aggregators without an integration", async () => {
```

File: apps/institution-service/src/shared/utils/permissionValidation.test.ts [hunk 2]
```diff
@@ -231,4 +233,105 @@ describe("permissionValidation", () => {
...
+              authorization: createTestAuthorization({
+                permissions: [UiUserPermissions.CREATE_AGGREGATOR_INTEGRATION],
+              }),
+            },
+          } as Request,
+        }),
+      ).toBe(true);
+    });
+
+    it("returns false if they are a super admin and they there aren't aggregators without an integration", async () => {
```
Comment (Craiting) [inline apps/institution-service/src/shared/utils/permissionValidation.test.ts:253]: Same typo with "they there"

File: apps/institution-service/src/shared/utils/permissionValidation.test.ts (hunk 1 above)
Comment (wesrisenmay-mx) [reply apps/institution-service/src/shared/utils/permissionValidation.test.ts]: Fixed

File: apps/institution-service/src/shared/utils/permissionValidation.test.ts (hunk 2 above)
Comment (wesrisenmay-mx) [reply apps/institution-service/src/shared/utils/permissionValidation.test.ts]: Fixed

**Review by Craiting — APPROVED** [review]

**Pull request merged**